src/components/Reactions/ReportActionItemReactions.js

```diff
@@ -81,12 +109,16 @@ const ReportActionItemReactions = (props) => {
                         key={reaction.emoji}
                     >
                         <EmojiReactionBubble
+                            ref={props.forwardedRef}
```

Comment:
✋ Coming from https://github.com/Expensify/App/issues/25976

We are preventing the popover from closing when clicking on the popover and the anchor. In this case, the entire View component where the reactions sit is used as the anchor, causing clicks to be ignored in this area.